src/codegen/codegen_acc_visitor.cpp

```diff
@@ -355,5 +359,12 @@ void CodegenAccVisitor::print_dt_update_to_device() const {
                       get_variable_name(naming::NTHREAD_DT_VARIABLE));
 }
 
+void CodegenAccVisitor::visit_protect_statement(const ast::ProtectStatement& node) {
```

Comment:
Why does this need to be overriden in `CodegenAccVisitor`?

Reply:
👍  - was a just leftover!